flow/project.py

```diff
@@ -1886,27 +2036,49 @@ def _print_progress(x):
 
                 t = time.time()
                 num_jobs = len(jobs)
-                statuses = []
+                label_results = []
                 for i, job in enumerate(jobs):
-                    statuses.append(_get_job_status(job))
+                    label_results.append(_get_job_labels(job))
                     if time.time() - t > 0.2:  # status interval
                         print(
-                            'Collecting job status info: {}/{}'.format(i+1, num_jobs),
+                            'Collecting job label info: {}/{}'.format(i+1, num_jobs),
                             end='\r', file=err)
                         t = time.time()
                 # Always print the completed progressbar.
-                print('Collecting job status info: {}/{}'.format(i+1, num_jobs), file=err)
-                return statuses
+                print('Collecting job label info: {}/{}'.format(i+1, num_jobs), file=err)
 
-    def _fetch_status_in_parallel(self, pool, pickle, jobs, ignore_errors, cached_status):
+        results, index, i = list(), dict(), 0
+
+        for job in jobs:
+            result = dict()
+            result['job_id'] = str(job)
+            result['operations'] = dict()
+            result['labels'] = list()
+            result['_operations_error'] = list()
+            result['_labels_error'] = list()
+            results.append(result)
```

Comment:
Here `result` and `results` are really close. This isn't a mistake, but it does make the code harder to read. Maybe `results_entry` would be more clear.